user.go

```diff
@@ -17,32 +19,61 @@ type usersRoot struct {
 	Meta  meta   `json:"meta"`
 }
 
+// SocialAccounts are social usernames or urls
+type SocialAccounts struct {
```

Comment:
There are some extra fields, also one for Google Plus, maybe a candidate for removal

```
 "social_accounts": {
  "github": "https://github.com/t0mk",
  "google": "https://plus.google.com/114835579379015959288",
  "keybase": "https://keybase.io/t0mk",
  "gravatar": "https://gravatar.com/t0mk88",
  "linkedin": "https://www.linkedin.com/in/t0mk88"
 },
```